### ISSUE TITLE
Customised font colours not working

### ISSUE THREAD

**Issue by reVoniaC** [notion-enhancer/notion-repackaged#50]
For a long time, I've been using a customised theme which I implemented by changing the colours in `AppData\Local\Programs\Notion Enhanced\resources\app\embedded_enhancer\mods\dark+\variables.css`. It's always worked fine. But recently, after trying out other themes, when I went back to dark+, the font colours became stuck to a certain theme (sorry I can't recognise which theme it is). The UI colours I defined in `dark+\variables.css` are still displaying correctly, though.

This is what the font colours now look like in Notion Enhanced when I use the dark+ theme:
![image](https://user-images.githubusercontent.com/20645394/137431122-a41b228f-e861-4e65-9133-a66c7a08cadc.png)

These are the colours I defined in `dark+\variables.css`, which are different from the colours in the screenshot above:
![image](https://user-images.githubusercontent.com/20645394/137431200-b20d109d-0b65-4528-8ac1-8f361399a237.png)

Any help would be much appreciated~ Thanks a lot!

**Comment by dragonwocky** [comment]
![image](https://user-images.githubusercontent.com/16874139/137432670-bf2551dc-27ee-4199-9cd8-a0ffc5d52865.png)

tldr: notion is changing their colours and its broken everything, nothing to do but wait

see https://github.com/notion-enhancer/notion-enhancer/issues/615 for issue tracking
